3party/sdf_image/CMakeLists.txt

```diff
@@ -1,5 +1,11 @@
 project(sdf_image)
 
+if (CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(
```

Comment:
I'm trying to disable as little as possible. This way for minizip is added only "-Wno-unused-value" and for sdf_image is added only "-Wno-shorten-64-to-32". If I create a macro I'll disable to much.